src/isIsogram.test.js

```diff
@@ -6,4 +6,21 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return true if word has no repeating letters`, () => {
+    expect(isIsogram('playgrounds')).toBe(true);
+  });
+
+  it(`should return false if word has repeating letters`, () => {
+    expect(isIsogram('look')).toBe(false);
+  });
+
+  it(`should return true if word has no repeating letters
+   and has capital letters`, () => {
+    expect(isIsogram('Playgrounds')).toBe(true);
+  });
+
+  it(`should return true if word is empty`, () => {
+    expect(isIsogram(' ')).toBe(true);
```

Comment:
```suggestion
    expect(isIsogram('')).toBe(true);
```

this is an empty string. `' '` - is not empty